internal/command/services/postgres/restart.go

```diff
@@ -66,17 +72,31 @@ func runRestart(ctx context.Context) error {
 	}
 
 	for _, machine := range machines {
```

Comment:
We may want to acquire leases on all machines before we begin the restart process.

Reply:
Yeah, I thought about that too.